src/frontend/src/sass/_variables.scss

```diff
@@ -11,6 +11,7 @@ $line-height: 1.5;
 $primary-blue: #036;
 $link-blue: #1a5a96;
 $secondary-yellow: #fcba19;
+$dodger-blue: #3b99fc;
```

Comment:
Is this colour used anywhere else? If not, maybe we should leave it out of our branding variables.

Reply:
so far, I only find we use it here. but I think we will use it later so i will put it in our branding variables

Reply:
Based on conversations with Jordan and Anthony, we will only be using colours and elements as defined by the Figma files under 'Components.' The UI/UX team has assured us that they will only be using styling as defined under that section.

Reply:
Please DO NOT add any colours that are not there in the branding component page.

Reply:
this color actually is focus, i just remove this one and use focus